packages/sitecore-jss-vue/src/components/Text.test.ts

```diff
@@ -75,6 +75,17 @@ describe('<Text />', () => {
     expect(rendered.element.innerHTML).toBe(props.field.value);
   });
 
+  it('should render number value', () => {
```

Comment:
Do we need to test negative numbers or zero value edge cases?

Reply:
Definitely we need a test for `zero` value.